webpack/assets/javascripts/foreman_settings.js

```diff
@@ -0,0 +1,14 @@
+import $ from 'jquery';
+
+$(document).ready(() => {
+  if (window.location.pathname === '/settings') {
```

Comment:
This will break if Foreman is mounted under a difference location. See the foreman_url function.

Reply:
you can use a regexp similiar to how its implemented for auth sources js at https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/foreman_auth_source.js#L81

Reply:
or use foreman_url instead.